src/uvw/util.hpp

```diff
@@ -507,21 +513,21 @@ struct Utilities {
 
         if(0 == uv_interface_addresses(&ifaces, &count)) {
             std::for_each(ifaces, ifaces+count, [&interfaces](const auto &iface) {
-                InterfaceAddress interface;
+                InterfaceAddress tInterface;
```

Comment:
Umm, `iface` is the name of a lambda parameter. Please suggest another name.
Also, I have a suggestion for this lambda (avoiding the need of a second name altogether): to use a constructor for `InterfaceAddress` taking a `const uv_interface_address_t &` (viz. `iface` here). We could benefit from using `std::vector<InterfaceAddress>::emplace_back`.